docs/bos/index.md

```diff
@@ -28,6 +28,7 @@ Build fully decentralized applications for all chains. Publish and get discovere
     <Feature url="https://near.social" title="NEAR Social" subtitle="The first BOS gate" image="near-social.png" />
     <Feature url="https://bos.gg" title="bos.gg" subtitle="Near Loves Ethereum" image="near-eth.png" />
     <Feature url="https://welldone-gateway.vercel.app/" title="WellDone Gateway" subtitle="A multichain gateway" image="welldone.png" />
+    <Feature url="https://twitter.com/TheblockchainOS/" title="TheBlockChainOS" subtitle="Discover The Activities on BOS through the BOS Community" image="bos-contact.png" />
```

Comment:
This is a twitter and not a BOS Gateway

Reply:
True... however while BOS Gateway curates the gateways 

We are a community built to create a space where we all can discuss, amplify and disect everything BOS through high quality simple  educational and engaging contents 

Via the platform,  

•partnerships and collaborations are being built 
•Teams and developers come to promote their content and events 
•Most of all we help facilitate the onboarding of developer and users into BOS

So it serves more than twitter handle, other features (coming) such as DiscoverBOS.org that helps with seamless onboarding, engagement and curation of BOS components, Communities,  products and activities.

It can can be seen as a merge of Nearweek and BOS Discover Page

Reply:
However that section is for BOS Gateways, which means that it lists BOS Gateways and not twitter accounts.

This begs the question: do we want to add twitter accounts? and if yes, then where?

Keeping the docs well organised is not a simple task, and if we randomly add underlated links to anywhere, then people will simply not find them.

I am open to suggestions on how to re-arange the docs to hold twitter handles, including renaming the section.

Reply:
@cenwadike heads up, we have re-modeled the landing pages, so maybe now it is easier to find a place for this?

Reply:
Thanks @gagdiez. I'll take a tour and make adjustments to the pull request